src/_gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -76,13 +80,13 @@ def ges_pflegev_beitr_satz_zusatz_kinderlos_dummy(
     return out
 
 
-@policy_info(start_date="2023-07-01", name_in_dag="ges_pflegev_beitr_satz")
-def ges_pflegev_beitr_satz_mit_kinder_abschlag(
+@policy_info(start_date="2023-07-01", name_in_dag="ges_pflegev_beitr_satz_arbeitnehmer")
+def ges_pflegev_beitr_satz_arbeitnehmer_mit_kinder_abschlag(
     anz_eig_kind_bis_24: int,
     ges_pflegev_zusatz_kinderlos: bool,
     sozialv_beitr_params: dict,
 ) -> float:
-    """Care insurance contribution rate.
+    """Employee's are insurance contribution rate.
```

Comment:
```suggestion
    """Employee's long term care insurance contribution rate.
```

The qualifier below should make clear this applies to the policy regime for the relevant dates active period (same elsewhere).